editoast/src/views/stdcm_search_environment.rs

```diff
@@ -106,7 +121,7 @@ impl From<StdcmSearchEnvironmentCreateForm> for Changeset<StdcmSearchEnvironment
         (status = 201, body = StdcmSearchEnvironment),
     )
 )]
-async fn overwrite(
+async fn create(
```

Comment:
Does this mean that we're going to create data that will never be deleted anymore?

Reply:
We also require an endpoint to list search environments. I suggest we do this in another PR.